package.json

```diff
@@ -17,14 +17,16 @@
     "connect-mongo": "^0.8.1",
     "cookie-parser": "~1.3.4",
     "debug": "~2.1.3",
+    "dotize": "^0.1.26",
     "express": "~4.12.2",
-    "express-handlebars": "~2.0.1",
+    "express-handlebars": "~3.0.0",
```

Comment:
Slight bump to the original handlebars package, but nothing new. I just like to bump packages I'm working with to keep things up to date. Also gives me slight hope that the problem I'm working with was fixed upstream :P